plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -227,10 +210,10 @@ func TestVMUpgradeBytesNetworkUpgradesWithGenesis(t *testing.T) {
 
 	// Get a json specifying a Network upgrade at genesis
 	// to apply as upgradeBytes.
-	subnetEVMTimestamp := time.Unix(10, 0)
+	subnetEVMTimestamp := big.NewInt(0)
```

Comment:
nit: can we use `common.Big0` for these?

Reply:
sure

Reply:
thanks